content/agent/_index.fr.md

```diff
@@ -52,7 +50,7 @@ L'Agent 6 est la dernière version majeure de l'Agent Datadog. La grande différ
 
 * [DogStatsD][6] peut être utilisé sur un socket unix plutôt que sur udp.
 
-* Personnalisez votre agent v6 et [DogStatsD][6] beaucoup plus facilement et avec beaucoup plus d'options de configuration, pour inclure ou exclure presque tout. Il y a aussi un agent «puppy» qui est installation vraiment minimale.
+* Personnalisez votre Agent v6 et [DogStatsD][6] beaucoup plus facilement et avec beaucoup plus d'options de configuration, pour inclure ou exclure presque tout. Il y a aussi un Agent «puppy» qui est installation vraiment minimale.
```

Comment:
One unit of [whitespace around _les guillemets_](http://www.la-ponctuation.com/guillemets.html). :)